Documentation/ExtensionArchitecture/FileStructure/Configuration/TCA/Index.rst

```diff
@@ -14,6 +14,14 @@ The folder :file:`EXT:my_extension/Configuration/TCA/` may contain or override
 All files in this directory are automatically included during the TYPO3
 :ref:`bootstrap <bootstrapping>`.
 
+..  versionadded:: 12.0
+    Files within :file:`Configuration/TCA/` files are loaded within a dedicated scope.
+    This means that variables defined in those files can't leak into the following files.
+
+    ..  note::
+        Until TYPO3 v12, variables declared in these files were in a shared scope,
```

Comment:
"Until TYPO3 v12" can be misunderstood. At first read, I assumed, in TYPO3 v12 the content has to be wrapped (when using local variables). But this seems not the case. So I suggest something like: "Until TYPO3 v11 and below" ..." to make it clearer.

Reply:
Ack